corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -152,15 +156,16 @@ hqDefine("app_manager/js/details/case_claim", function () {
                     searchProperties[i].name,
                     label,
                     appearance,
+                    default_value,
                     propItemSet
-                ));
+                ))
             }
         } else {
-            self.searchProperties.push(searchProperty('', '', '', itemSet()));
+            self.searchProperties.push(searchProperty('', '', '','', itemSet()));
```

Comment:
There are a getting to be a lot of parameters here. A (totally optional) improvement would be to make `searchProperty` expect a single object and verify the expected options are present, something like [this](https://github.com/dimagi/commcare-hq/blob/d3c2eb371e5f26947ef0e0341650634d77c6c4d0/corehq/apps/users/static/users/js/mobile_workers.js#L221-L228), which then makes calls look more like [this](https://github.com/dimagi/commcare-hq/blob/d3c2eb371e5f26947ef0e0341650634d77c6c4d0/corehq/apps/users/static/users/js/mobile_workers.js#L506-L512), kind of like using named kwargs in python.

Reply:
I really like this idea-- just added in my lastest commit (last change unless errors arise)